### PR TITLE
Sort companies alphabetically

### PR DESCRIPTION
**Pull request by benbarnett**

### CODE REVIEW

**Comment by darokel** [comment]
:+1:

File: app/helpers/debt_management_helper.rb [hunk 1]
```diff
@@ -0,0 +1,5 @@
+module DebtManagementHelper
+  def companies_list
+    t('companies.companies').sort { |a, b| a <=> b }
```
Comment (esbel) [inline app/helpers/debt_management_helper.rb]: `.sort { |a, b| a <=> b }` is equal to just `.sort`

Comment (benbarnett) [inline app/helpers/debt_management_helper.rb]: Ah right cool, I'll remove then

Comment (benbarnett) [inline app/helpers/debt_management_helper.rb]: Removed

Comment (darokel) [inline app/helpers/debt_management_helper.rb]: Good call

**Comment by esbel** [comment]
thanks :+1:

**Pull request merged**